NorthstarDLL/masterserver.cpp

```diff
@@ -814,595 +692,385 @@ void MasterServerManager::AuthenticateWithServer(char* uid, char* playerToken, c
 	requestThread.detach();
 }
 
-void MasterServerManager::AddSelfToServerList(
-	int port, int authPort, char* name, char* description, char* map, char* playlist, int maxPlayers, char* password)
+void MasterServerManager::WritePlayerPersistentData(const char* playerId, const char* pdata, size_t pdataSize)
 {
-	if (!Cvar_ns_report_server_to_masterserver->GetBool())
```

Comment:
add this again

Reply:
Seems to be working again, according to @RoyalBlue1 ^^